docs/content/en/docs/Getting started/_index.md

```diff
@@ -40,6 +40,71 @@ See [Image Matrix Support](/docs/reference/image_matrix) for additional supporte
 
 {{% /alert %}}
 
+
+## Checking artifacts signatures
+
+{{% alert title="Note" color="warning" %}}
+
+This feature will be available in Kairos version `1.5.0` and in all future releases.
+
+{{% /alert %}}
+
+Our ISO releases have a sha256 files to checksum the validity of the artifacts. At the same time our sha256 files are signed automatically in the CI during the 
+release workflow to verify that they haven't been tampered with, adding an extra step on the supply chain. 
+
+It is recommended that before starting any installation the whole security chain is validated by verifying our sha256 signature and validate the checksum matches with the download artifacts.
+
+
+To validate the whole chain you would need:
+
+1. `sha256sum` which is usually installed by default on most linux distributions.
+2. `cosign` to verify the signatures of the sha256 file. You can install cosign via their [installation docs](https://docs.sigstore.dev/cosign/installation/)
+3. ISO, sha256, certificate and signature files for the release/flavor that you want to verify. All the artifacts are available on the [kairos release page](https://github.com/kairos-io/kairos/releases)
+
+
+In this example we will use the `v1.5.1` version and `opensuse-leap` flavor
+
+First we check that we have all needed files:
+
+```bash
+$ ls      
+kairos-opensuse-leap-v1.5.1.iso         kairos-opensuse-leap-v1.5.1.iso.sha256.pem
+kairos-opensuse-leap-v1.5.1.iso.sha256  kairos-opensuse-leap-v1.5.1.iso.sha256.sig
+```
+
+We first verify that the sha256 checksums haven't been tampered with:
+
+```bash
+$ COSIGN_EXPERIMENTAL=1 cosign verify-blob --cert kairos-opensuse-leap-v1.5.1.iso.sha256.pem --signature kairos-opensuse-leap-v1.5.1.iso.sha256.sig kairos-opensuse-leap-v1.5.1.iso.sha256 
```

Comment:
isn't there a cosign container that we can suggest to use instead?

Reply:
Indeed I think there is. In any case, the closing install instructions are linked which means that there is a variety of install options provided to the user, so they can choose to use whatever they want. This one here is just an example, I can send a  PR to change the example to use the docker image instead